.github/workflows/test-client.yml

```diff
@@ -69,6 +69,24 @@ jobs:
           echo "NETWORK=${{ inputs.network }}" >> $GITHUB_ENV
           echo "AGGREGATOR_ENDPOINT=${{ inputs.aggregator_endpoint }}" >> $GITHUB_ENV
           echo "GENESIS_VERIFICATION_KEY=$(curl -s ${{ inputs.genesis_verification_key }})" >> $GITHUB_ENV
+          echo "TRANSACTIONS_HASHES_TO_CERTIFY=${{ inputs.transactions_hashes_to_certify }}" >> $GITHUB_ENV
+
+      - name: Prepare environment variable for Cardano transactions aggregator capability (ubuntu-22.04, macos-12)
+        id: aggregator_capability_unix
+        if: matrix.os == 'ubuntu-22.04' || matrix.os == 'macos-12'
```

Comment:
Prefer to use the [`runner.os` context variable](https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context) as it does not include the version.

This should be the default case by using a "negative" check: `runner.os != 'Windows'` (don't forget to update the name of the steps).